### PR TITLE
DBZ-3290 Oracle preparations from Incubating to Stable

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-3290

### CODE REVIEW

**Comment by gunnarmorling** [comment]
LGTM; waiting with the merge until after our go/no-go call tomorrow.

>

**Comment by gunnarmorling** [comment]
Remove hard-coded version "0.9.0.Alpha1" in listings

**Comment by Naros** [comment]
@gunnarmorling I rebased the `oracle.adoc` against master and I do see that all `scn` and `commit_scn` values are quoted as strings so we likely need to see why the website build isn't reflecting that for master.

**Comment by gunnarmorling** [comment]
> we likely need to see why the website build isn't reflecting that for master.

I'm not sure whether a push to the "documention" dir in _debezium_ triggers a website build. @ani-sha would know.

**Comment by Naros** [comment]
Logged [DBZ-3317](https://issues.redhat.com/browse/DBZ-3317) to align decimal mappings for decimal.handling.mode.  I did check and we do support all 3 modes as Jiri indicated in both `SnapshotDatatypesIT` and `StreamingDatatypesIT` tests.  I also closed [DBZ-807](https://issues.redhat.com/browse/DBZ-807) as it's no longer relevent.

**Comment by Naros** [comment]
The `LONG` data-type is another one of those weird ones where its currently fully supported with LogMiner; however it is not yet supported by XStream because the data is delivered using the chunk-processing callbacks.  I've updated [DBZ-2948](https://issues.redhat.com/browse/DBZ-2948) to capture also including support for `LONG`.

**Comment by ani-sha** [comment]
> I'm not sure whether a push to the "documention" dir in debezium triggers a website build.

@gunnarmorling @Naros So we have actually missed that. There is no trigger for website build upon any action to `documentation` dir.

**Comment by gunnarmorling** [comment]
Ah, thx, @ani-sha. Would be a nice improvement for sure.

File: documentation/modules/ROOT/pages/connectors/oracle.adoc [hunk 1]
```diff
@@ -1950,6 +1778,200 @@ The message does not contain the update to the schema, but the complete new sche
 
 |===
 
+[[oracle-xstreams-support]]
+== XStreams support
+
+The {prodname} Oracle connector by default ingests changes using native Oracle LogMiner.
+The connector can be toggled to use Oracle XStream instead and to do so specific database and connector configurations must be used that differ from that of LogMiner.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/oracle.adoc:1785]: The GG license requirement should be mentioned here.

File: documentation/modules/ROOT/pages/connectors/oracle.adoc [hunk 2]
```diff
@@ -1950,6 +1778,200 @@ The message does not contain the update to the schema, but the complete new sche
...
+
+In addition, supplemental logging must be enabled for captured tables or the database in order for data changes to capture the _before_ state of changed database rows.
+The following illustrates how to configure this on a specific table, which is the ideal choice to minimize the amount of information captured in the Oracle redo logs.
+
+[source,indent=0]
+----
+ALTER TABLE inventory.customers ADD SUPPLEMENTAL LOG DATA (ALL) COLUMNS;
+----
+
+=== Creating Users for the connector
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/oracle.adoc:1816]:
```suggestion
=== Creating users for the connector
```

**Comment by Naros** [comment]
Copyright changed in PR https://github.com/debezium/debezium.github.io/pull/651

**Comment by Naros** [comment]
* Logged [DBZ-3327](https://issues.redhat.com/browse/DBZ-3327) to align default values for other connectors.

**Comment by jpechane** [comment]
@Naros Hi, it seems it is complete now. Could you please rebase to remove the convflict?

File: documentation/modules/ROOT/pages/connectors/oracle.adoc [hunk 3]
```diff
@@ -708,9 +701,9 @@ To make this possible, {prodname}'s Oracle connector always follows the _delete_
 [[oracle-data-type-mappings]]
 == Data Type mappings
 
-The Oracle conenctor represents changes to rows with events that are structured like the table in which the rows exists. The event contains a field for each column value. How that value is represented in the event depends on the Oracle data type of the column. The following sections describe how the connector maps oracle data types to a _litearl type_ and a _semantic type_ in event fields.
+The Oracle connector represents changes to rows with events that are structured like the table in which the rows exists. The event contains a field for each column value. How that value is represented in the event depends on the Oracle data type of the column. The following sections describe how the connector maps oracle data types to a _litearl type_ and a _semantic type_ in event fields.
```
Comment (jpechane) [inline documentation/modules/ROOT/pages/connectors/oracle.adoc:704]:
```suggestion
The Oracle connector represents changes to rows with events that are structured like the table in which the rows exists. The event contains a field for each column value. How that value is represented in the event depends on the Oracle data type of the column. The following sections describe how the connector maps oracle data types to a _literal type_ and a _semantic type_ in event fields.
```

**Comment by jpechane** [comment]
@Naros This is good to go. I've fixed one more typo oversight. Could you please just rebase on the latest master? Feel free to merget it yourself after that.

**Pull request merged**